Endorser.md

```diff
@@ -0,0 +1,61 @@
+# Transaction Endorser Support
+
+Note that the ACA-Py transaciton support is in the process of code refactor and cleanup.  The following documents the current state, but is subject to change.
+
+ACA-Py supports an [Endorser Protocol](https://github.com/hyperledger/aries-rfcs/pull/586), that allows an un-privieged agent (an "Author") to request another agent (the "Endorser") to sign their transactions so they can write these transactions to the ledger.
+
+Transaction Endorsement is built into the protocols for Schema, Credential Definition and Revocation, and endorsements can be explicitely requested, or ACA-Py can be configured to automate the endorsement workflow.
+
+## Setting up Connections between Authors and Endorsers
+
+Since endorsement involves message exchange between two agents, these agents must establish and configure a connection before any endorsements can be provided or requested.
+
+Once the connection is established and `active`, the "role" (either Author or Endorser) is attached to the connection using the `/transactions/{conn_id}/set-endorser-role` endpoint.  For Authors, they must additionally configure the DID of the Endorser as this is required when the Author signs the transaction (prior to sending to the Endorser for endorsement) - this is done using the `/transactions/{conn_id}/set-endorser-info` endpoint.
```

Comment:
I think it should be possible to have the DID specified on the command line.

Reply:
Agree this is a command line parameter but not currently used (the auto-connection-setup support is still TBD)